assets/apps/customizer-controls/src/radio-buttons/RadioButtonsComponent.js

```diff
@@ -152,14 +154,23 @@ const RadioButtonsComponent = ({ control }) => {
 	};
 	const { label, large_buttons, showLabels } = control.params;
 
+	const options = getChoices();
+	if (custom) {
+		options.custom = {
+			tooltip: __('Import custom SVG', 'neve'),
```

Comment:
One more thing, I think that we can use `Custom SVG` here as well to limit the translation impact. I think that the tooltip doesn't show up either way 👀.